.github/workflows/build-test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
-      max-parallel: 2
+      max-parallel: 1
```

Comment:
I guess that would impact the test performance overall (not just dataplex related tasks). Are we keeping track of this, so in the future it can be increased again?

Reply:
ok, understood!